docker/images/Dockerfile.quickstart

```diff
@@ -0,0 +1,10 @@
+# This file is intended for development only.
+# It will allow users to quickly get this app running so that they can start making updates without worrying about
+# whether or not dependencies are properly installed.
+FROM node:14
+
+WORKDIR /home/app
+
+RUN npm install -g npm@7
+
+CMD npm run install-dependencies && npm run develop -- -H 0.0.0.0 2>&1
```

Comment:
[?] Do we need to start the dev server based on `${PORT}`?

[suggest] Consider moving the command to docker-compose file rather than baked into the image.

Reply:
Because PORT is being passed in from the `.env.development` file, Gatsby picks it up automatically. I can move the command into docker compose and add a port flag for clarity.

Reply:
Didn't know that `gatsby` picked up the `${PORT}` env var automatically. That works.

Dealer's choice on `CMD` whether it lives in Dockerfile or compose file. If you decide to move to compose, I would move `expose` there as well and keep the image generic.